index.js

```diff
@@ -23,4 +23,8 @@ function fastFreeze(config, secureChar, logger){
 
 }
 
-module.exports = fastFreeze;
\ No newline at end of file
+if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
```

Comment:
Do we need this change? I mean I've used *browserify* in past but I've never changed the source code for that.

Reply:
I believe after this change, we can even this file directly in the browser instead of using any external tool.